Slim/Formats/XML.pm

```diff
@@ -416,8 +416,18 @@ sub parseRSS {
 		
 		$feed{'image'} = $url;
 	}
-	elsif ( $xml->{'itunes:image'} ) {
-		$feed{'image'} = $xml->{'itunes:image'}->{'href'};
+	elsif ( ref $xml->{'channel'}->{'itunes:image'} ) {
```

Comment:
Shouldn't we leave the old code in? After all it's dealing with another potential use case, right?

Reply:
Agree. I'll amend accordingly.